doc/denite.txt

```diff
@@ -1451,12 +1451,64 @@ abbr		(String)			(Optional)
 ==============================================================================
 CREATE KIND						*denite-create-kind*
 
-Note: Not implemented yet...
+Custom Kind can be created to contain extra actions for your custom source.
+Before create your new kind, you should have a look at existing kinds in
+rplugin/python3/denite/kind/*.py
+
+Instead of create new kind from scratch, you can extend one of the existing
+Kind class, let's take `openable` for example: >
+
+	from ..kind.openable import Kind as Openable 
+
+	class Kind(Openable):
+		def __init__(self, vim):
+			super().__init__(vim)
+			self.name = 'mykind'
+
+		def action_open(self, context):
+			target = context['targets'][0]
+			path = target['action__path']
+			" assume you have a MyEdit command
+			self.vim.call('MyEdit', path)
+>
+After create new Kind class, alias it to your source by >
+
+	self.kind = Kind(vim)
+<
+in the {__init__} method of your source class.
```

Comment:
Vim's help already use `{xxxxx}` syntax for an argument so it should not be used.